packages/devextreme/js/__internal/ui/html_editor/modules/m_toolbar.ts

```diff
@@ -311,6 +312,9 @@ if (Quill) {
 
         return this._getToolbarItem(buttonItemConfig);
       }
+      if (item.name && this._isAcceptableItem(item.widget, 'dxDropDownButton')) {
```

Comment:
If you leave the code as it is, any element containing a widget field with the value 'dxDropDown' will be configured with the above instructions.